app/views/companies/show.html.haml

```diff
@@ -5,7 +5,8 @@
         %li.panel-body
           %div.col-md-12
             %div.col-md-3
-              %img.img-rounded.img-responsive{src: "http://placehold.it/500x500"}
+              -# %img.img-rounded.img-responsive{src: "http://placehold.it/500x500"}
+              %image_tag @company.avatar.url
```

Comment:
Make sure to remove the "%" at the beginning of the image_tag and add a "=" instead. Also add this to every image for the companies and internships that belong to that company.